src/args.js

```diff
@@ -2,8 +2,8 @@ import yargs from 'yargs';
 
 export default yargs
   .usage('Auth0 Deploy CLI')
-  .option('verbose', {
-    alias: 'v',
+  .option('debug', {
```

Comment:
I was under impression that there is already a debug option somewhere, do we really want to convert this?

Reply:
we have the `yargs` framework in place and it has useful features, e.g., `--help`.  however, we do need to check whether `--debug` is true through the `yargs` api, so I have updated.